RoslynPluginGenerator/AnalyzerPluginGenerator.cs

```diff
@@ -185,6 +259,20 @@ private string CreateBaseWorkingDirectory()
             return baseDirectory;
         }
 
+        private bool HasAnalyzers(IPackage package, string language)
+        {
+            return GetAnalyzers(package, language).Any();
+        }
```

Comment:
The method is private; tests won't be able to see it.
This method also means you're effectively calling _GetAnalyzers_ twice times for each package. This isn't efficient (although efficiency is not a huge concern here).
What does the log output look like?